src/Examples/UnitModels/cstr_testing.ipynb

```diff
@@ -0,0 +1,1123 @@
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"water\"].fix(200)\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"sulfuric_acid\"].fix(0.334)  # calculated from 0.9 wt% SA in stream\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"].fix(1e-5)\n",
+    "m.fs.M101.catalyst_feed.temperature.fix(298.15)\n",
+    "m.fs.M101.catalyst_feed.pressure.fix(1e5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fixing unit model specifications\n",
+    "\n",
+    "Now that we have fixed our inlet feed conditions, we will now be fixing the operating conditions for the unit models in the flowsheet. Let us initialize H101 with an outlet temperature of 328.15 K. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.H101.outlet.temperature.fix(328.15)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll add constraints defining the reactor volume and conversion in relation to the stream properties. Particularly, we want to use our CSTR performance relation:  \n",
+    "\n",
+    "$V = \\frac{v_0 X} {k(1-X)}$, where the CSTR reaction volume $V$ will be specified, the inlet volumetric flow $v_0$ is determined by stream properties, $k$ is calculated by the reaction package, and $X$ will be calculated. Reactor volume is commonly selected as a specification in simulation problems, and choosing conversion is often to perform reactor design.\n",
+    "\n",
+    "We will assume that 80% of our total reactor volume is working volume (CSTR reaction volume). For the CSTR, we have to define the conversion in terms of ethylene oxide as well as the CSTR reaction volume. This requires us to create new variables and constraints relating reactor properties to stream properties.  Note that the CSTR reaction volume variable (m.fs.R101.volume) does not need to be defined here since it is internally defined by the CSTR model. Additionally, the heat duty is not fixed, since the heat of reaction depends on the reactor conversion (through the extent of reaction and heat of reaction)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.R101.total_volume = Var(initialize=6.9225, bounds=(0, 15))  # m^3\n",
+    "m.fs.R101.conversion = Var(initialize=0.80, bounds=(0, 1))  # fraction\n",
+    "\n",
+    "m.fs.R101.conv_constraint = Constraint(\n",
+    "    expr=m.fs.R101.conversion*m.fs.R101.inlet.\n",
+    "    flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"] ==\n",
+    "    (m.fs.R101.inlet.flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"] -\n",
+    "     m.fs.R101.outlet.flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"]))\n",
+    "\n",
+    "m.fs.R101.vol_constraint = Constraint(\n",
```

Comment:
A comment on why 80% is used here might be useful (there might be one that I missed).